### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by VOID404**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

#37

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,17 +59,48 @@ metadata:
...
 
-8. Deploy.
+7. Deploy.
 
 ```bash
 make deploy
 ```
 ## Usage
 
+Compass Manager watches for Kyma Custom Resource changes, and when a Kyma with `application-connector` component is created, it registers the Kyma runtime in the Compass Director and creates a Compass Manager Mapping with the ID assigned by the Compass Director.
```
Comment (mvshao) [inline README.md:77]:
```suggestion
Compass Manager watches for Kyma Custom Resource changes, and when a Kyma with `application-connector` module is created, it registers the Kyma runtime in the Compass Director and creates a Compass Manager Mapping with the ID assigned by the Compass Director.
```

File: README.md [hunk 2]
```diff
@@ -93,16 +115,22 @@ make deploy
...
 > **TIP:** `CompassManagerMappings` created with dry run are labeled `kyma-project.io/cm-dry-run: Yes`
 
-> Explain how to use the project. You can create multiple subsections (H3). Include the instructions or provide links to the related documentation.
-
 ## Development
 
-> Add instructions on how to develop the project or example. It must be clear what to do and, for example, how to trigger the tests so that other contributors know how to make their pull requests acceptable. Include the instructions or provide links to related documentation.
+To run the project locally, use the following command:
+```shell
+make build
```
[the remark this answers is not in the document]
Comment (mvshao) [reply README.md:122]: PEBKAC

File: README.md [hunk 3]
```diff
@@ -68,17 +59,48 @@ metadata:
...
 
-8. Deploy.
+7. Deploy.
 
 ```bash
 make deploy
 ```
 ## Usage
 
+Compass Manager watches for Kyma Custom Resource changes, and when a Kyma with `application-connector` module is created, it registers the Kyma runtime in the Compass Director and creates a Compass Manager Mapping with the ID assigned by the Compass Director.
```
Comment (grego952) [inline README.md:77]:
```suggestion
Compass Manager watches for Kyma custom resource changes. When Kyma with the Application Connector module is created, it registers Kyma runtime in the Compass Director and creates a Compass Manager Mapping with the ID assigned by the Compass Director.
```

File: README.md [hunk 4]
```diff
@@ -68,17 +59,48 @@ metadata:
...
+It then configures the Compass Runtime Secret on the client cluster.
```
Comment (grego952) [inline README.md:78]:
```suggestion
It then configures the Compass runtime Secret on the client cluster.
```
Unless the Compass Runtime Manager is some type of component.

**Pull request merged**